packages/compass-e2e-tests/helpers/commands/add-collection.ts

```diff
@@ -31,6 +33,29 @@ export type AddCollectionOptions = {
   encryptedFields?: string;
 };
 
+async function waitForAnimations(
+  browser: CompassBrowser,
+  element: Element<'async'>
+): Promise<void> {
+  let previousResult = {
+    ...(await element.getLocation()),
+    ...(await element.getSize()),
+  };
+  await browser.waitUntil(async function () {
+    // small delay to make sure that if it is busy animating it had time to move
+    // before the first check and between each two checks
+    await browser.pause(50);
+
+    const result = {
+      ...(await element.getLocation()),
+      ...(await element.getSize()),
+    };
+    const stopped = _.isEqual(result, previousResult);
+    previousResult = result;
+    return stopped;
+  });
+}
```

Comment:
Do we want to move this into the file https://github.com/mongodb-js/compass/blob/93307133a16a7320bd205cd04424424bbbadaaa1/packages/compass-e2e-tests/helpers/commands/wait-for-animations.ts
Maybe we can reduce the duplication there too.